packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.component.js

```diff
@@ -108,8 +108,8 @@ export class CheckoutOrderSummary extends PureComponent {
         }
 
         const label = coupon_code ? __('Coupon code discount') : __('Discount');
-
-        return this.renderPriceLine(-Math.abs(discount_amount), label);
+        const mods = { couponCode: coupon_code };
```

Comment:
mods should be used only for passing mods that will passed as mods in child component and used for styling. I see that data is being unpacked from mods instead. If you just need to pass coupon code, use another props for this.
https://github.com/scandipwa/scandipwa/blob/ecfcce4bd9d66a6fda31d846ba7be362c830ab2d/packages/scandipwa/src/component/CheckoutOrderSummaryPriceLine/CheckoutOrderSummaryPriceLine.component.js#L60